index.html

```diff
@@ -10,7 +10,16 @@
     href="https://use.fontawesome.com/releases/v5.4.1/css/all.css">
     <link rel='stylesheet' href='assets/css/style.css' type='text/css'>
     <!-- Bootstrap CDN-->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" 
+      integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+      <style>
+        @media screen and (max-width: 550px) {
+          #randSec h2 {
```

Comment:
This does not target the text inside the slides.
You can accomplish this with:
```
@media screen and (max-width: 550px) {
          .rev_div > p {
            font-size: 22px;
          }
        }
```
which targets the children of all all divs of class `.rev_div` (which is where all the comments go).
However, after testing the above change, the comment slide looks much better!
I will add these changes to main.css